### DISCUSSION TITLE
Sanni: Do you take donations?

### DISCUSSION THREAD

**Discussion by Ancyker** [sanni/cartreader#438] [General Chat]
I noticed that there's no mention of donations (Patreon, "buy me a coffee", etc) but I also searched and didn't see anyone asking either. Are you not accepting donations or has just no one asked before?

**Comment by sanni** [comment]
I have not spend any thoughts on that yet. So far the best way to contribute is to just build and use the Open Source Cart Reader and report errors or post suggestions and maybe even add a few lines of code. That has worked exceptionally well in the past years. 😃

**Reply by Ancyker** [reply]
Of course :)

If you don't want to accept donations then do you have a preferred charity or cause you'd like to direct to instead?
